src/main/java/seedu/address/logic/commands/person/ViewCommand.java

```diff
@@ -16,9 +16,9 @@
  */
 public class ViewCommand extends Command {
     public static final String COMMAND_WORD = "view";
-    public static final String MESSAGE_VIEW_CONTACT_SUCCESS = "Viewing contact: %1$s";
+    public static final String MESSAGE_VIEW_PERSON_SUCCESS = "Viewing contact: %1$s";
```

Comment:
Think we agreed previously that the messages should be from the perspective of the app. Here and else models, `Viewing contact` should be `Displayed contact: %1$s`. 

Also, I think list command should be `Listed all contacts!` if it is not already. Clear should be `Cleared all contacts!`.

Reply:
To be fixed by #182